wiki/Guides/Adding_Custom_Hitsounds/en.md

```diff
@@ -2,41 +2,56 @@
 
 From the thread: *[how to add custom hitsound? by: neonat](https://osu.ppy.sh/community/forums/posts/3215699)* (and modified)
 
-## Getting the custom hitsounds
+## Custom Hitsound Files
 
-You'll first need some custom hitsounds! You can either:
-- You can get them from the [Custom Hitsound Library](../Custom_Hitsound_Library/),
-- or make your own
-  - if you do make your own, try to keep the format as `.wav` - as it is best for looping.
+In order to use custom hitsound files in osu!, you'll first need some custom hitsounds! The [Custom Hitsound Library](../Custom_Hitsound_Library/) is a great resource for existing sound samples, including cymbals, drums, bells, whistles and more! Alternatively, if you can't find what you're looking for, you can make your own samples!
 
-Once you get them, move them into the song folder itself. If you don't know where your song folder is located, follow these directions:
+Hitsounds should typically be in the `.wav` or `.ogg` format, as these have no playback delay and loop correctly. `.mp3` formatted sound files have slight delays and may not accurately match up to the sounds in the music you are trying to hitsound. However, they but may still be useable for certain special hitsounds such as applause or ambient noises, where the size of `.wav` or `.ogg` files may be prohibitively large.
+
+Once you have your desired sound files, they should be moved into the song folder of the beatmap you would like to use them in. If you don't know where your song folder is located, follow these directions:
 1. Open osu!
-2. Edit
+2. Select the *Edit* option
```

Comment:
This method of reaching the song folder not work on osx builds afaik, a different process is needed:
1. Right click on the Osu! application icon and click "Show Package Contents"
2. drive_c -> osu! -> Songs -> find the song folder 
(Sorting by "Last Modified" is a good idea)

Reply:
Was not aware of this, clarified with some MacOS users and have added additional steps here as you suggested.